src/background.js

```diff
@@ -20,7 +28,7 @@ function runShowMeTheCode(tabId, tab) {
     };
 
     if (isGitHub()) {
```

Comment:
Dismissed by https://github.com/Viridovix/show-me-the-code-github/pull/33#commitcomment-15086508